### DISCUSSION TITLE
Expose sqflite's batch api?

### DISCUSSION THREAD

**Discussion by mochar** [cachapa/sqlite_crdt#6] [General]
I want to do batch inserts as a single atomic unit. For this I need `db.batch()` which `crdt` does not seem to expose. Can this be made available if it isn't too much trouble? Thanks again !

**Comment by cachapa** [comment]
Any reason to prefer batches to transactions? Batches are committed as transactions anyway.

I understand that batches incur slightly less overhead, but for the majority of cases it's probably negligible.

**Reply by mochar** [reply]
Indeed to reduce overhead, but I am not very knowledgeable on this matter so I will take your word for it and use transactions. Thanks a lot!

**Reply by cachapa** [reply]
Sorry if my reply sounded dismissive.

I think it does make sense to implement batches, just wanted to know the use case.

It might make sense to create a feature ticket to track this.

**Reply by mochar** [reply]
No i am very thankful for your comment because i genuinely didn't know that the difference is not that significant for most use cases. I am happy using transactions instead! And thanks again for the great package.

**Reply by cachapa** [reply]
I've created issue #7 to track this.

**Comment by cachapa** [comment]
Released with 3.0.0
